mojo/core/data_pipe_consumer_dispatcher.cc

```diff
@@ -393,13 +397,25 @@ DataPipeConsumerDispatcher::Deserialize(const void* data,
 
 #if defined(CASTANETS)
   base::subtle::PlatformSharedMemoryRegion::ScopedPlatformHandle region_handle;
+  auto guid = base::UnguessableToken::Deserialize(state->buffer_guid_high,
+                                                  state->buffer_guid_low);
+  if (state->options.flags & MOJO_CREATE_DATA_PIPE_FLAG_GUID_SHM) {
+    base::ScopedFD fd;
+    base::FilePath path;
+    if (base::CreateNamedSharedMemory(guid.ToString(),
```

Comment:
I think the function name is enough because this function creates NamedSharedMemory.
This function has the option of creating shared memory from the existing file without creating new file.

Reply:
I agree. CreateNamedSharedMemory is enough.